src/System.Runtime.Extensions/tests/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
@davidsh When you say to leave "locked" as true, do you mean for every project.lock.json or only for the one in System.Runtime.Extensions?